<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemName</class>
 <widget class="QDialog" name="SystemName">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1307</width>
    <height>909</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_hospital">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>医院名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_device">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>290</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设备名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_print">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>400</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>打印机：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_print">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>410</y>
     <width>451</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>便携式打印机</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>台式打印机</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_hospital">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>170</y>
     <width>521</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_device">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>290</y>
     <width>521</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>600</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/confirmbt.png);
</string>
   </property>
   <property name="text">
    <string>  确定</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>600</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/btexit.png);</string>
   </property>
   <property name="text">
    <string>   返回</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
